package.json

```diff
@@ -54,7 +54,8 @@
     "@heroicons/react": "1.0.6",
     "@hookform/resolvers": "2.8.10",
     "@identity.com/sol-did-client": "3.1.4",
-    "@marinade.finance/marinade-ts-sdk": "2.0.9",
+    "@marinade.finance/marinade-ts-sdk": "5.0.4",
```

Comment:
I think we should use only the ts-sdk now

Reply:
👍🏼